crates/dojo-core/src/resource_metadata.cairo

```diff
@@ -136,13 +136,14 @@ impl ResourceMetadataIntrospect<> of dojo::database::introspect::Introspect<Reso
 #[starknet::contract]
 mod resource_metadata {
     use super::ResourceMetadata;
+    use super::RESOURCE_METADATA_MODEL;
```

Comment:
```suggestion
    use super::RESOURCE_METADATA_SELECTOR;
```
Was done out of this PR, but if we could rename it could be great for comprehension. :+1: